opt/Makefile

```diff
@@ -77,6 +77,9 @@ device selection options (fetch, build, and test):
   GPU=1         # build for GPU
   CUDA=1        # build for GPU
   JETSON=1      # build for Nvidia Jetson
+
+enterprise variant
```

Comment:
I assume you wanted this line to be a comment?

Reply:
this is a comment in `define HELP` scope